tools/provenance-generator/README.md

```diff
@@ -0,0 +1,72 @@
+# Knative Provenance Generator
+
+As part of achieving SLSA Compliance, we need to generate [provenance](https://slsa.dev/provenance/v0.2).
+
+Kubernetes project already has a library that can generate provenances but we will wrap it with some go code that can pull our build info from Prow.
+
+We will pull the following from Prow jobs:
+
+- https://prow.knative.dev/view/gs/knative-prow/logs/nightly_kn-plugin-quickstart_main_periodic/1552222987300638720
```

Comment:
I don't understand what this first thing is.

Reply:
It is the .env file in testdata. When this tool runs in the CI it will pull info from the environment, the typical values are posted at that link. 

Need to write better docs.